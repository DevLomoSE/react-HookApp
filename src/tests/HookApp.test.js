import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe('Test for the component <HookApp />', () => {
    test('should rendar and match with the snapshot', () => {
        const wrapper = shallow(<HookApp />);

        expect(wrapper).toMatchSnapshot();
    });
});