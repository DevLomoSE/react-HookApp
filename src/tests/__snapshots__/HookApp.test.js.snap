// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the component <HookApp /> should rendar and match with the snapshot 1`] = `
<div>
  HookApp
</div>
`;
